src/controllers/accommodationController.js

```diff
@@ -7,51 +7,54 @@ import imageUploader from '../helpers/imageUploader';
 
 cloudinary.config({ CLOUDINARY_URL: process.env.CLOUDINARY_URL });
 
+const {
+  CREATED, NO_INFO_YET, RETRIEVED, NO_ACCOMMODATION
+} = strings.accommodation.success;
+
 export default class AccommodationController {
   static async createAccommodation(req, res) {
     const {
-      name, description, location, availableSpace, cost, highlights, amenities
+      name, description, locationId, availableSpace, cost, currency, highlights, amenities
     } = req.body;
 
     const images = await imageUploader(req, res);
 
     const accommodation = {
       name: name.toLowerCase(),
       description,
-      location: location.toLowerCase(),
+      locationId,
       availableSpace,
       cost,
+      currency: currency.toUpperCase(),
       highlights,
       amenities,
       owner: req.user.payload.id,
       images
     };
-
     try {
       const newAccommodation = await models.accommodations.create(accommodation);
 
-      return responseUtil(res, 201, strings.accommodation.success.CREATED, newAccommodation);
+      return responseUtil(res, 201, CREATED, newAccommodation);
     } catch (error) { return responseError(res, 400, error); }
   }
 
-  static async getMyAccommodations(req, res) {
-    const owner = req.user.payload.id;
-
-    const MyAccommodations = await models.accommodations.findAll({ where: { owner } });
+  static async getAllAccommodations(req, res) {
+    const { role, id: owner } = req.user.payload;
+    if (role === 5) {
+      const MyAccommodations = await models.accommodations.findAll({ where: { owner } });
```

Comment:
Include the location object when fetching accommodations so that you have something like this in the response  for the value of location 
``` location:{
id:1,
name: kampala
} ```